webrtc.html

```diff
@@ -336,25 +336,25 @@
           class="simple">
             <tbody>
               <tr>
-                <th colspan="2">Enumeration description</th>
+                <th colspan="2">Enumeration description (non-normative)</th>
```

Comment:
I tried that, but the hyperlink ends up looking ugly on the blue background: [screenshot](http://i.imgur.com/5xbWaC9.png)

Also, the correct section of JSEP is already linked a couple lines above this, so it's a bit redundant anyway.

Any suggestions? My goal is to make it clear that the enum names are normative, but the meanings are not.